mysql-test/suite/innodb/t/instant_alter_debug.test

```diff
@@ -664,4 +664,15 @@ WHERE variable_name = 'innodb_instant_alter_column';
 
 SET GLOBAL innodb_stats_persistent = @save_stats_persistent;
 
+CREATE TABLE t1(f1 INT, f2 TEXT)ENGINE=InnoDB;
```

Comment:
I think that the `SET GLOBAL` is part of the after-test cleanup. I would disable the persistent statistics also for this test, to avoid any interference from automatic recalculation.